src/py/flwr/common/constant.py

```diff
@@ -37,7 +37,16 @@
     TRANSPORT_TYPE_VCE,
 ]
 
-SUPEREXEC_DEFAULT_ADDRESS = "0.0.0.0:9093"
+# Addresses
+# SuperNode
+CLIENTAPPIO_API_DEFAULT_ADDRESS = "[::]:9094"
+# SuperExec
+EXEC_API_DEFAULT_ADDRESS = "[::]:9093"
+# SuperLink
+DRIVER_API_DEFAULT_ADDRESS = "[::]:9091"
+FLEET_API_GRPC_RERE_DEFAULT_ADDRESS = "[::]:9092"
+FLEET_API_GRPC_BIDI_DEFAULT_ADDRESS = "[::]:8080"
+FLEET_API_REST_DEFAULT_ADDRESS = "[::]:9093"
```

Comment:
Let's continue to use the same defaults as before, and not switch the IPv4 ones to IPv6. We defaulted to IPv6 in earlier versions of Flower, but it caused some unforeseen issues.